std/parallelism.d

```diff
@@ -4553,8 +4553,12 @@ version (unittest)
 // tons of stuff and should not be run every time make unittest is run.
 version (parallelismStressTest)
```

Comment:
I ran this stress test and found out, that it was not run for ages (lot's of bugs). So I tried to fix it as good as possible.